tests/DevBetterWeb.Tests/Core/Entities/MemberTests/BookHelpers.cs

```diff
@@ -0,0 +1,20 @@
+﻿using DevBetterWeb.Core.Entities;
+
+namespace DevBetterWeb.Tests.Core.Entities.MemberTests
+{
+    public static class BookHelpers
+    {
+        public static Book GetDefaultBook()
```

Comment:
This is good to use for tests instead of newing up a book. I'd make it more clear this is for tests somehow in the name, ideally, like GetDefaultTestBook or GetTestBook etc. Could also be an opportunity to use AutoFixture.
https://github.com/AutoFixture/AutoFixture/wiki/Cheat-Sheet#complex-type